internal/binutils/testdata/build_binaries.go

```diff
@@ -65,11 +65,15 @@ func main() {
 		}
 
 	case "windows":
+		// Many gcc enviroments may create binaries that trigger false-positives in anti-viruses.
```

Comment:
nit: could you keep comments in 80 columns? That's the style the rest / most of the code in pprof uses.